Header header                       # Header.
float32 force                       # Force.
float32[] force_dis                 # Force distribution.